src/lib/achievements/AchievementsManager.ts

```diff
@@ -0,0 +1,45 @@
+import * as statsfm from '../../interfaces/statsfm';
+import Manager from '../Manager';
+
+export default class AchievementsManager extends Manager {
+  /**
+   * @description Get a achievement by ID.
+   * @param {number} id The ID of the achievement.
+   * @returns {Promise<Achievement>} Returns a promise with a single {@link Achievement}.
+   */
+  async get(id: number): Promise<statsfm.Achievement> {
+    const res = await this.http.get(`/achievements/${id}`);
+
+    return res.data.item as statsfm.Achievement;
+  }
+
+  /**
+   * @description Get a list of achievements by IDs.
+   * @param {string} ids The IDs of the achievements
+   * * @returns {Promise<Achievement[]>} Returns a promise with a {@link Achievement}s.
+   */
+  async list(ids: number[]): Promise<statsfm.Achievement[]> {
+    const res = await this.http.get<statsfm.Achievement[]>(`/achievements/list`, {
```

Comment:
We don't use /list for listing all, just /achievements would be following our spec :)